Odata-docs/TOC.yml

```
@@ -352,6 +352,10 @@
         href: /odata/connectedservice/t4-templates
       - name: Settings reference
         href: /odata/connectedservice/reference
+  - name: OData Cli
```

```suggestion
  - name: OData CLI
```
I suggest using **CLI** instead of **Cli** when referring to the tool in "natural" language (i.e. outside of code contexts), since it's an abbreviation